js/audiOrbits.js

```diff
@@ -270,6 +468,43 @@ var audiOrbits = {
 		// set Cursor for "fixed" parallax mode
 		if (sett.parallax_option == 3) self.positionMouseAngle(sett.parallax_angle);
 
+		// Regen levels to see the effect of the setting change sooner.
+		if (reGenLevels && !reInitFlag) {
+			// Re-adjust remaining settings and update UI to reflect the changes.
+			// Unfortunately its not possible to change Wallpaper Engine settings
+			// (that I am aware of), so user has to manage the validity of each
+			// setting to ensure they add up to 1.
+			// Disabled for now because it doesn't recalc correctly for large
+			// datasets.
+			if (self.state !== 0 && wewwApp.IsInit() && false) {
+				self.recalcAttractProb();
+				self.PushSettingsToUi();
```

Comment:
As you can see, both of these functions are disabled atm. The "recalcAttractProb()" is broken with a lot of attractors it has to recalculate. It worked fine when I had 5.